sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -672,12 +709,12 @@ aggregates AS (
     mozfun.stats.mode_last(
       ARRAY_AGG(previous_build_id ORDER BY submission_timestamp)
     ) AS previous_build_id,
-    UNIX_DATE(DATE(SAFE.TIMESTAMP(ANY_VALUE(subsession_start_date)))) - ANY_VALUE(
+    MAX(UNIX_DATE(DATE(SAFE.TIMESTAMP(subsession_start_date)))) - MAX(
       profile_creation_date
     ) AS profile_age_in_days,
     FORMAT_DATE(
       "%F 00:00:00",
-      SAFE.DATE_FROM_UNIX_DATE(ANY_VALUE(profile_creation_date))
+      SAFE.DATE_FROM_UNIX_DATE(MAX(profile_creation_date))
```

Comment:
This doesn't have to be `MAX`, but something deterministic is better than `ANY_VALUE` and I wanted to avoid `mozfun.stats.mode_last` because it's expensive in terms of query complexity that would probably better spent on new fields.